### PR TITLE
Create neonpunk theme to reviews.

### PR DESCRIPTION
**Pull request by gannerr**
Honestly, I just wanted to play with animations! Experimented with text-shadows, animation, and opacity
- Visual features:
     - Make neon pink glowing animations, like a flickering lightbulb of sorts.
     - Make submit button glow up upon hover
     - Make "Most Recent Reviews" button create shadow upon hover
     - Make "Delete Reviews" button have pulsing animation upon hover

### CODE REVIEW

**Comment by gannerr** [comment]
Screenshot 2020-07-01 at 2 53 28 PM
Screenshot 2020-07-01 at 2 53 18 PM

File: portfolio/src/main/webapp/css/reviews.css [hunk 1]
```diff
@@ -15,15 +16,28 @@
...
-  border: 1px solid #ccc;
-  border-radius: 4px;
   resize: none;
+  align-content: center;
+  background-color: transparent;
+  border: 3px solid #00d7c3;
+  border-radius: 50px;
+  -webkit-transition: all .15s ease-in-out;
+  transition: all .15s ease-in-out;
+  color: #00d7c3;
```
Comment (terencedignon) [inline portfolio/src/main/webapp/css/reviews.css:26]: nit alphabetized

File: portfolio/src/main/webapp/css/reviews.css [hunk 2]
```diff
@@ -123,3 +130,188 @@ label {
...
+  }
+  100% {
+    color: #ffe6ff;
+    text-shadow: 0 0 0.6rem #ffe6ff, 0 0 1.5rem #ff65bd,
+      -0.2rem 0.1rem 1rem #ff65bd, 0.2rem 0.1rem 1rem #ff65bd,
+      0 -0.5rem 2rem #ff2483, 0 0.5rem 3rem #ff2483;
+  }
+}
+
+@keyframes flicker {
```
Comment (terencedignon) [inline portfolio/src/main/webapp/css/reviews.css:193]: could you add a comment describing these definitions?

File: portfolio/src/main/webapp/reviews.html [hunk 3]
```diff
@@ -36,38 +36,41 @@
...
         </form>
       </div>
+    </section>
 
-      <label for="max-comments"># of Comments (between 1 and 5):</label>
+    <section class="section section-dark">
+      <div class="sign">
+        <span class="fast-flicker">S</span>e<span class="flicker">e</span>!
+      </div>
+      <br><br><br>
```
Comment (terencedignon) [inline portfolio/src/main/webapp/reviews.html:66]: Usually a code smell if you need more than one br or p. Could we use margin here instead?

**Review by terencedignon — CHANGES_REQUESTED** [review]

**Comment by terencedignon** [comment]
WOW this looks amazing. Had a few comments, but looking forward to seeing it in action.

**Review by terencedignon — APPROVED** [review]
Thanks., LGTM

**Pull request merged**